qiskit_machine_learning/algorithms/classifiers/pegasos_qsvc.py

```diff
@@ -340,14 +345,15 @@ def precomputed(self) -> bool:
     @precomputed.setter
     def precomputed(self, precomputed: bool):
         """Sets the pre-computed kernel flag. If ``True`` is passed then the previous kernel is
-        cleared. If ``False`` is passed then a new instance of ``QuantumKernel`` is created."""
+        cleared. If ``False`` is passed then a new instance of
+        :class:`~qiskit_machine_learning.kernels.BaseKernel` is created."""
```

Comment:
Should this be FidelityQuantumKernel - that is a BaseKernel, but is that too abstract that a user needs to know the actual type?

Reply:
That's right, updated.